pkg/conversions/gatewayroute_types_conversion_test.go

```diff
@@ -435,6 +544,245 @@ func TestConvert_CRD_GRPCGatewayRouteAction_To_SDK_GrpcGatewayRouteAction(t *tes
 	}
 }
 
+func TestConvert_CRD_GatewayRouteHostnameMatch_To_SDK_GatewayRouteHostnameMatch(t *testing.T) {
+	type args struct {
+		crdObj *appmesh.GatewayRouteHostnameMatch
+		sdkObj *appmeshsdk.GatewayRouteHostnameMatch
+	}
+	tests := []struct {
+		name       string
+		args       args
+		wantSDKObj *appmeshsdk.GatewayRouteHostnameMatch
+	}{
+		{
+			name: "normal case",
```

Comment:
Same comment as above w.r.t adding test cases for all the supported options. Let's name the test cases appropriately -> "exact match case" for example

Reply:
Looking for those additional UT cases.